src/Provider/FreeGeoIp/FreeGeoIp.php

```diff
@@ -33,13 +34,20 @@
     private $baseUrl;
 
     /**
+     * @var string
+     */
+    private $locale;
+
+    /**
      * @param HttpClient $client
+     * @param string     $locale
      * @param string     $baseUrl
      */
-    public function __construct(HttpClient $client, string $baseUrl = 'https://freegeoip.net/json/%s')
+    public function __construct(HttpClient $client, string $locale = 'en', string $baseUrl = 'https://freegeoip.net/json/%s')
```

Comment:
Is "en" a good default value? 

In `getRequest()` you do a check if it is null or not.